model/src/main/java/com/basistech/rosette/apimodel/AdmRequest.java

```diff
@@ -48,12 +49,25 @@
      */
     private final LanguageCode language;
 
+
+    public AdmRequest(String profileId,
+                      AnnotatedText text,
+                      O options,
+                      LanguageCode language) {
+        super(profileId);
+        this.text = text;
+        this.options = options;
+        this.genre = null;
+        this.language = language;
+    }
+
+
     @Builder     // workaround for inheritance https://github.com/rzwitserloot/lombok/issues/853
     public AdmRequest(String profileId,
```

Comment:
Same question as the other class; should this entire method be flagged with `@Deprecated` ?